config/rbac/role.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   creationTimestamp: null
-  name: manager-role
```

Comment:
Could you remind me why this change is needed ?

Reply:
changed back.